pyproject.toml

```
@@ -98,7 +100,7 @@ markers = [
     "integration_test: Marks a test as an integration test, which is often slow (deselect with '-m \"not integration_test\"')",
     "workflow_test: Marks a test as a complete pixelator workflow, which is extremely slow (deselect with '-m \"not workflow_test\"')"]
 
-addopts = ["-p no:pytest-workflow", "-m not workflow_test", "--benchmark-disable"]
+addopts = ["-p no:pytest-workflow", "-m not workflow_test", "--benchmark-disable", "--mpl", "--mpl-results-path=tests/reports_mpl", "--mpl-generate-summary=html",]
```

I think it is preferable to keep the commandline short and use the INI options as much as possible:

```suggestion
mpl-results-path = "tests/reports_mpl"
mpl-generate-summary = "html"
addopts = ["-p no:pytest-workflow", "-m not workflow_test", "--benchmark-disable", "--mpl"]
```